src/modules/filelist/icons/FileThumbnail.tsx

```
@@ -163,46 +164,38 @@ const FileThumbnail: React.FC<FileThumbnailProps> = ({
-        />
-      </BadgeKonnector>
-    )
+    return <BadgeKonnector file={file}>{fileIcon}</BadgeKonnector>
   }
 
+  const infected = isInfected(file)
+
+  const fileIconWithInfection = infected ? (
+    <Badge
+      size="large"
+      badgeContent={<Icon icon="warning-circle" color="var(--errorColor)" size={20} />}
```

ESLint

[error] Replace `<Icon·icon="warning-circle"·color="var(--errorColor)"·size={20}·/>` with `⏎········<Icon·icon="warning-circle"·color="var(--errorColor)"·size={20}·/>⏎······`

(prettier/prettier)